### PR TITLE
Yo 39

### PR DESCRIPTION
**Pull request by joe-yeager**
This is functionally done, but it still needs a little love in the styling department, but I left that for when we fine tune as a group.    If you would like to test this, you can either update a user of your liking so they have the `isAdmin:true` in rethinkdb, or you can pull the latest version of wafflecone, and then log in as the admin user.  The admin user is admin@froyo.com and the password is welovefroyo

### CODE REVIEW

**Comment by domshyra** [comment]
#peer

File: app/views/createCarpoolView.html [hunk 1]
```diff
@@ -33,6 +33,6 @@ <h2 class="form-signup-heading">Create a Carpool</h2>
     </div>
 
     <div ng-show="carpoolExists" class="help-inline2"> Error: Cannot create carpool. This carpool already Exists.</div><br>
-    <button type="submit" class="btn btn-primary btn-lg btn-inline" ng-click="submitted=true; createCarpool(createform.$invalid)">Create Carpool</button>
```
Comment (domshyra) [inline app/views/createCarpoolView.html:36]: Does this actually make a difference?

Comment (joe-yeager) [inline app/views/createCarpoolView.html:36]: Oh, I see what you are talking about, I edited the wrong file, oops.

**Comment by ladinu** [comment]
#peer

**Pull request merged**